### PR TITLE
Added recipe for zodbpickle

### PR DESCRIPTION
**Pull request by pmlandwehr**
Because of how this package is set up, a bunch of `py27` files will fail when built on `py3k`, and a bunch of `py3k` files will fail when built on `py27`. This seems rather inefficient, but I'm not sure of a great fix.

### CODE REVIEW

File: recipes/zodbpickle/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - setuptools
```
Comment (jakirkham) [inline recipes/zodbpickle/meta.yaml]: It needs `setuptools` at runtime?

File: recipes/zodbpickle/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+test:
+  imports:
+    - zodbpickle
+    - zodbpickle.tests
+
+about:
+  home: http://pypi.python.org/pypi/zodbpickle
+  license: ZPL 2.1
+  license_file:
```
Comment (jakirkham) [inline recipes/zodbpickle/meta.yaml]: 😱 I think we missed something.

File: recipes/zodbpickle/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  preserve_egg_dir: True
+  number: {{ build_num }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
```
Comment (jakirkham) [inline recipes/zodbpickle/meta.yaml]: Please add `toolchain` as it has compiled code.

File: recipes/zodbpickle/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    - setuptools
+
+test:
+  imports:
+    - zodbpickle
+    - zodbpickle.tests
+
+about:
+  home: http://pypi.python.org/pypi/zodbpickle
+  license: ZPL 2.1
```
Comment (jakirkham) [inline recipes/zodbpickle/meta.yaml]: Appears to be dual licensed: `PSF 2 and ZPL 2.1`.

File: recipes/zodbpickle/meta.yaml (hunk 2 above)
Comment (pmlandwehr) [inline recipes/zodbpickle/meta.yaml]: Added a link, but not sure if the manifest includes it. We'll see...

File: recipes/zodbpickle/meta.yaml (hunk 1 above)
Comment (pmlandwehr) [inline recipes/zodbpickle/meta.yaml]: Removed run req...

**Comment by jakirkham** [comment]
Thanks @pmlandwehr.

**Pull request merged**